src/bindings/capi/tvgCapi.cpp

```diff
@@ -528,7 +528,7 @@ TVG_EXPORT Tvg_Result tvg_gradient_set_spread(Tvg_Gradient* grad, const Tvg_Stro
 TVG_EXPORT Tvg_Result tvg_gradient_get_spread(Tvg_Gradient* grad, Tvg_Stroke_Fill* spread)
 {
     if (!grad) return TVG_RESULT_INVALID_ARGUMENT;
-    *spread = (Tvg_Stroke_Fill) reinterpret_cast<Fill*>(grad)->spread();
+    if (spread) *spread = (Tvg_Stroke_Fill) reinterpret_cast<Fill*>(grad)->spread();
```

Comment:
I think it returns error when spread is valid.... -> TVG_RESULT_INVALID_ARGUMENT?

Reply:
Please check now whether it is what you meant.
I also unified the file formatting - not sure whether I was allowed to do it is this pr...